src/reactapp/src/components/CheckoutForm/CheckoutFormWrapper.jsx

```diff
@@ -36,50 +33,59 @@ function CheckoutFormWrapper({ initialData, children }) {
     }
 
     const { cart, customer } = initialData;
-    const email = _get(cart, 'email', '');
+    const email = _get(cart, 'email') || '';
     const appliedCoupon = _get(cart, 'appliedCoupon');
```

Comment:
@rajeev-k-tomy for my understanding, coupon is the only value here that is not required, and that's why it's the only one that does not have a fallback value?

Reply:
Your assumption is right @wigman